include/libopencm3/swm050/gpio.h

```diff
@@ -106,11 +125,19 @@ void gpio_input(uint16_t gpios);
 void gpio_output(uint16_t gpios);
 void gpio_sel_af(uint16_t gpios, bool af_en);
 void gpio_pullup(uint16_t gpios, bool en);
-void gpio_in_en(uint16_t gpios, bool en);
+void gpio_input_en(uint16_t gpios, bool en);
```

Comment:
I know it's a new file, but changing apis should generally be accompanied with the explanation somewhere for it.

Reply:
It was changed to distinguish it more from the interrupt enable function.  Should I note that in the doxygen or just change it back?

Reply:
Fixed.